tests/README.md

```diff
@@ -0,0 +1,114 @@
+# ROSA CLI Function Verification Testing
+This package is the automation package for Function Verification Testing on the ROSA CLI. 
+
+## Structure of tests
+```sh
+tests
+|____e2e      
+    |____e2e_suite_test.go                        ---- test suite for all the e2e tests
+    |____...                                      ---- other tests organized in domain files
+|____utils
+|    |____common                                  ---- package which contains common methods 
+|    |    |____constants                          ---- package for all constants
+|    |____config                                  ---- package which contains tests configuration methods 
+|    |____exec                                    ---- exec contains the different services to run the commands
+|    |    |____rosacli                            ---- ROSA CLI specific services/commands
+|    |____log                                     ---- tests logger
+|       
+|____prow_ci.sh
+```
+
+## Contibute to ROSA CLI tests
+
+Please read the structure and contribute code to the correct place
+
+<!-- ### Contribute to day1 case
+
+* Define profile in ./tests/ci/data/profiles folder
+* Define prepare steps according to the profile in rosa/tests/ci/profile_handler.go
+
+#### Day1 cases only do creation step, we have to define a day1 case to verify the day1 works well
+
+* Create the case in rosa/tests/e2e/<feature name>_test.go
+* Label the case with ***CI.Day1Post***
+* Label the case with importance ***CI.Critical***
+* Don't need to run creation step, just in BeforeEach step call function  ***PrepareRHCSClusterByProfileENV()*** it will load the clusterID prepared
```

Comment:
`PrepareRHCSClusterByProfileENV` need to be updated.

Reply:
gonna set it up

Reply:
updated with `config.GetCLusterID()` for now